spec/spec_helper.rb

```diff
@@ -1,3 +1,31 @@
+require 'simplecov'
+require 'simplecov_small_badge'
+
+SimpleCov.start(:rails) do
+  SimpleCov.formatters = SimpleCov::Formatter::MultiFormatter.new([
+    SimpleCov::Formatter::HTMLFormatter,
+    SimpleCovSmallBadge::Formatter,
+  ])
+end
+SimpleCov.minimum_coverage(65)
+SimpleCovSmallBadge::Formatter.class_eval do
```

Comment:
This monkey patch is very useful without this gem fails ,  can see context here: https://github.com/MarcGrimme/simplecov-small-badge/issues/15

Reply:
Ohh, thank u very much, knowing these kind of stuff helps a lot!